jobs/curator/templates/config/bpm.yml.erb

```diff
@@ -0,0 +1,8 @@
+processes:
+  - name: curator
+    hooks:
+      pre_start: /var/vcap/jobs/curator/bin/pre-start
+    ephemeral_disk: true
+    additional_volumes:
+    - path: /var/vcap/jobs/opensearch/config
+      writable: true
```

Comment:
Why is this here? Is this job reading from the opensearch job config? If not we should remove this

Reply:
We needed to pass it the certs, i can try it without

Reply:
It looks like the certs come from the `curator` job now?

https://github.com/cloud-gov/opensearch-boshrelease/blob/7f883cbc0e9ad85ec82e957f02c3cab7c144e8f3/jobs/curator/templates/config/config.yml.erb#L7-L9